server/server/urls.py

```diff
@@ -24,6 +26,16 @@
     path("logout/", LogoutView.as_view(), name="logout"),
     *HTMXAutoComplete.url_dispatcher("ac"),
     path("", include(cpho_urls)),
+    path(
+        "graphiql/",
+        csrf_exempt(api_views.GraphiQLView.as_view()),
+        name="graphiql",
+    ),
+    path(
+        "graphql/",
+        csrf_exempt(api_views.GraphQLView.as_view()),
+        name="graphql",
+    ),
```

Comment:
I moved these from cpho/urls to server/urls, it's cleaner if cpho/urls only covers the data-collection app